axelor-human-resource/src/main/java/com/axelor/apps/hr/service/expense/ExpenseLineServiceImpl.java

```diff
@@ -176,4 +180,16 @@ public String computeProjectTaskDomain(ExpenseLine expenseLine) {
                 .collect(Collectors.toList()))
         + ")";
   }
+
+  @Override
+  public List<Employee> getEmployeeDomain(LocalDate expenseDate) {
+    return Beans.get(EmployeeRepository.class)
+        .all()
+        .filter(
+            "self.user.blocked = false AND self.hireDate <= :expenseDate AND (self.leavingDate=null OR self.leavingDate >= :expenseDate) AND (self.user.expiresOn is null OR self.user.expiresOn> CURRENT_DATE) \n"
+                + "AND self.mainEmploymentContract.payCompany IN :companySet")
+        .bind("expenseDate", expenseDate)
+        .bind("companySet", AuthUtils.getUser().getCompanySet())
+        .fetch();
+  }
```

Comment:
This should be placed in a Employee related service like EmployeeFetchService with a more specific method name like getInvitedCollaborators . Also don't use Beans.get and use constructor injection.